CMakeLists.txt

```diff
@@ -153,6 +153,17 @@ target_add_torchvision(cv_segmentation)
 target_add_loguru(cv_segmentation)
 target_add_httplib(cv_segmentation)
 target_add_mavsdk(cv_segmentation)
+
+# cv_classification
+add_executable(cv_classification ${SOURCES} "tests/integration/cv_classification.cpp")
+target_add_json(cv_classification)
+target_add_matplot(cv_classification)
+target_add_opencv(cv_classification)
+target_add_torch(cv_classification)
+target_add_torchvision(cv_classification)
+target_add_loguru(cv_classification)
+target_add_httplib(cv_classification)
+target_add_mavsdk(cv_classification)
```

Comment:
Can this be moved to `tests/integration/CMakeLists.txt`? I think your branch is out of sync with main since we recently moved all the integration tests there. But it's all good. You'll just have to merge the updated main into this branch and move this CMake snippet there.